Remove reference to le_max_left.

Julian opened the pull request:
At least on the version of Lean currently in the leanpkg.toml,
library_search seems to directly suggest le_abs_self.

PatrickMassot commented:
Indeed! Could you rather modify the source in `src/solutions` and then run the exercise creation script before committing?

Julian committed:
Use regular ol' stdlib regular expression engine.

Julian committed:
Remove reference to le_max_left.

At least on the version of Lean currently in the leanpkg.toml,
library_search seems to directly suggest le_abs_self.

Julian committed:
Regenerate exercises.

Julian commented:
Ah, didn't notice that, yep fixed now to do that instead.

Julian commented:
Man, I somehow doubly screwed this up when making the new change. Sending a new PR to fix, sorry.
